networkx/algorithms/connectivity/kcutsets.py

```diff
@@ -39,9 +39,9 @@ def all_node_cuts(G, k=None, flow_func=None):
         computed. Default value: None.
 
     flow_func : function
-        Function to perform the underlying flow computations. Default value
-        edmonds_karp. This function performs better in sparse graphs with
-        right tailed degree distributions. shortest_augmenting_path will
+        Function to perform the underlying flow computations. Default value is
+        :meth:`edmonds_karp`. This function performs better in sparse graphs with
+        right tailed degree distributions. :meth:`shortest_augmenting_path` will
```

Comment:
This is the right idea, but unfortunately sphinx isn't smart enough to resolve references that originate in another module (at least not without a lot of extra help).

To get the linking working, you'll need to specify the *full location* of the object in the networkx package. If you prefix it with a `~` though, then only the method name will be shown in the html output (though the docstring itself definitely gets uglier!):

```suggestion
        :func:`~networkx.algorithms.flow.edmonds_karp`. This function performs
        better in sparse graphs with right tailed degree distributions.
        :func:`~networkx.algorithms.flow.shortest_augmenting_path` will
```

Reply:
Ok, That makes sense! Thanks for the reviews @rossbar!!